packages/generator/src/generator.ts

```diff
@@ -76,33 +78,38 @@ export async function generate(options: GeneratorOptions): Promise<void> {
  * TODO: Actually split the parsing of options between CLI and programmatic options. Currently users have to provide everything.
  */
 export async function generateWithParsedOptions(
-  options: GeneratorOptions
+  options: ParsedGeneratorOptions
```

Comment:
[q] If I got it right are the parsedOptions: All options required and the deprecated omitted. So I guess the TODO could go to the `generate` function - or could you make all things beside input/output optional anyhow?

Reply:
I'll just remove it, this was added before the other ticket was opened.